numpy/core/tests/test_regression.py

```diff
@@ -2333,6 +2333,18 @@ def test_void_item_memview(self):
         #del va
         #assert_equal(x, b'\x00\x00\x00\x00')
 
+    def test_reduce_warns(self):
```

Comment:
this test should probably be in the testminmax of numpy/core/tests/test_umath.py

and we have to hope it fpu exceptions work on all platforms properly

Reply:
fixed. 

One of the  CI test platforms is a VM without SSE, so passing this test there means this will warn whenever there is a NAN in the output of a min/max reduce. Note the code in loop.c.src to check that condition.